ada/utils.py

```diff
@@ -340,6 +340,7 @@ def upload_edc_media(report, study_id, record_id, field_id, token):
     study_data_point = "study-data-point"
     field_id = field_id
     path = posixpath.join(study_id, record, record_id, study_data_point, field_id)
+    report_name = record_id + "_" + "report"
```

Comment:
This will not have the `.json` extension and record_id might have something not fit for a file name, so we should probably clean it.